content/02.introduction.md

```diff
@@ -98,6 +98,23 @@ They also found that survivors continued to shed the virus for a median of 20 da
 - How can we evaluate the human response to SARS-CoV-2 on a molecular level?
 - What can omics technologies tell us about changes in the body following SARS-CoV-2 infection?
 
+SARS-CoV-2 is a relatively novel coronavirus, being third after SARS-CoV and MERS-CoV, causing severe pneumonia in humans [@doi:10.1016/bs.aivir.2018.01.001]. 
+Therefore, our knowledge of the molecular mechanisms by which SARS-CoV-2 infect the cells, is mostly obtained from studies on previously identified coronaviruses. 
+The entry of coronaviruses into a target cell is mediated by the binding of the viral spike (S) protein to a specific cellular receptor and subsequent priming/cleavage of S protein by cellular proteases [@pmid:16139595;@pmid:32272173]. 
+Angiotensin-Converting Enzyme 2 (ACE2) and Transmembrane Serine Protease 2 (TMPRSS2) have been identified as a prime receptor and a critical protease for SARS-CoV/CoV-2 entry into a target cell, respectively [@pmid:14647384; @pmid:32075877; @pmid:20926566; @pmid:32142651; @pmid:21325420]. 
+This has steered to a leading hypothesis on the role of ACE2 and TMPRSS2 expression in the SARS-CoV-2 infected cells, tissues and organs.
```

Comment:
```suggestion
This finding has led to a hypothesized role for ACE2 and TMPRSS2 expression in determining which cells, tissues and organs are most likely to be infected by SARS-CoV-2.
```
I think this is what that sentence is saying, but I'm not quite clear what you mean by the "leading hypothesis" -- if I missed the mark with this suggestion, please just let me know!